### PR TITLE
add signature field to fix ethers

### LINKED ISSUE #571: yParity mismatch for signed transactions
Ethers throws transactions signed with Interceptor

https://github.com/ethers-io/ethers.js/blob/main/lib.esm/crypto/signature.js#L296C1-L296C96

### PR DESCRIPTION
**Pull request by KillariDev**
fix https://github.com/DarkFlorist/TheInterceptor/issues/571

### CODE REVIEW

**Comment by KillariDev** [comment]
let's wait for ethers reply on this, we might get a fast fix

File: app/ts/simulation/services/SimulationModeEthereumClientService.ts [hunk 1]
```diff
@@ -566,18 +566,17 @@ export const getSimulatedBlockNumber = async (ethereumClientService: EthereumCli
...
 	for (const [index, simulatedTransaction] of simulationState.simulatedTransactions.entries()) {
 		if (hash === simulatedTransaction.signedTransaction.hash) {
-			const v = 'v' in simulatedTransaction.signedTransaction ? simulatedTransaction.signedTransaction.v : (simulatedTransaction.signedTransaction.yParity === 'even' ? 0n : 1n)
 			const additionalParams = {
 				blockHash: getHashOfSimulatedBlock(),
 				blockNumber: await ethereumClientService.getBlockNumber(),
 				transactionIndex: BigInt(index),
 				data: simulatedTransaction.signedTransaction.input,
-				v : v,
+				signature: 378356n, // signature field to fix ethers. TODO, remove when ethers is fixed
```
Comment (MicahZoltu) [inline app/ts/simulation/services/SimulationModeEthereumClientService.ts:579]: This constant should probably be in hex, and having it be 65 bytes would also help add clarity to what exactly it is.

Also, where is this constant from?  Does 65 0 bytes not work?

Comment (KillariDev) [reply app/ts/simulation/services/SimulationModeEthereumClientService.ts:579]: its "ethers"
3=e
7=t
8=h
5=r
6=s

**Comment by KillariDev** [comment]
fixed upstream in ethers

**Pull request closed without merging**